keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/batch/ContainerJobConfig.kt

```diff
@@ -0,0 +1,123 @@
+package com.netflix.spinnaker.keel.titus.batch
+
+import com.netflix.spinnaker.keel.titus.batch.ImageVersionReference.*
+
+const val RUN_JOB_TYPE : String = "runJob"
+
+/**
+ * Containers can be identified by either:
+ *  tag (e.g., "latest")
+ *  digest (e.g., "sha256:780f11bfc03495da29f9e2d25bf55123330715fb494ac27f45c96f808fd2d4c5")
+ */
+enum class ImageVersionReference {
+  TAG,
+  DIGEST
+}
+
+/**
+ * Holds the config info needed to execute a container via the Run Job stage
+ */
+data class ContainerJobConfig(
```

Comment:
Is it possible to rearrange this data class, so that all mandatory fields without default values would be on top?

Reply:
done!